longevity_test.py

```diff
@@ -125,7 +125,15 @@ def test_custom_time(self):
         if alternator_port:
             endpoint_url = 'http://{}:{}'.format(normalize_ipv6_url(self.db_cluster.nodes[0].external_address),
                                                  alternator_port)
-            alternator_create_table(endpoint_url, test_params=self.params)
+
+            with self.cql_connection_patient(self.db_cluster.nodes[0]) as session:
+                session.execute("""
+                    INSERT INTO system_auth.roles (role, salted_hash) VALUES (%s, %s)
+                """, (self.params.get('alternator_access_key_id'),
+                      self.params.get('alternator_secret_access_key')))
+
+            dynamodb_primarykey_type = self.params.get('dynamodb_primarykey_type', default='HASH')
```

Comment:
perhaps we can take alternator-related default values from the general defaults file of sct. or have a new file for alternator default. or there is already a place for that ? @fruch ?

Reply:
yes we should remove default from the code, we could put this in test_defaults